src/Localizers.jl

```diff
@@ -89,6 +90,34 @@ http://arxiv.org/abs/2105.11341
 """
 struct SECFisher <: LocalizationMethod end
 
+
+"""
+    SECNice{FT <: Real} <: LocalizationMethod
+
+Sample error correction as of Morzfeld, Vishny et al. (2024).
+Correlations are shrinked by a factor determined by correlation and ensemble size.
+The factors are automatically determined by a discrepancy principle.
+Thus no algorithm parameters are required, though some tuning of the discrepancy principle tolerances are made available.
+
+# Fields 
+
+$(TYPEDFIELDS)
+
+"""
+struct SECNice{FT <: Real, AV <: AbstractVector} <: LocalizationMethod
+    "number of samples to approximate the std of correlation distribution (default 1000)"
+    n_samples::Int
+    "scaling for discrepancy principle for ug correlation (default 1.0)"
+    tol_ug::FT
```

Comment:
Can you please rename `tol_ug` to `δ_ug` and `tol_gg` to `δ_gg` for consistency?

Reply:
Done